<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Convoy</class>
 <widget class="QMainWindow" name="Convoy">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convoy - 0.9 - Val Veirs - Beam Reach</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="runModel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ship Source Level File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFile">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>670</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>../shipSourceLevels.df</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>670</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R Dataframe file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveRdata">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>630</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save R data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFileName">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>670</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>../modelOutput.df</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="nYears">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="dbBackgroundSigma">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dbBackground">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Background dB level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of years in simulation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+-</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yMid">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
    <property name="value">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xMid">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>610</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
    <property name="value">
     <double>110.000000000000000</double>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>571</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yMax">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yMin">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>590</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xMax">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>610</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
    <property name="value">
     <double>130.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xMin">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>610</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
    <property name="value">
     <double>90.000000000000000</double>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Plot</string>
    </property>
    <widget class="QRadioButton" name="graphY_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Probability</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="graphY_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Cumulative Probability</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="nBins">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_52">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>49</width>
       <height>11</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>N bins</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>203</width>
      <height>179</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Number of ships </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Ships/day</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>per year</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="shipsPerDay">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="N_ships_0">
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>9000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>AIS Ships</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="N_ships_1">
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Future Tanker</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSpinBox" name="N_ships_2">
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Future Bulker</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="N_ships_3">
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Future Container</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum acoustic range +-</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxRange">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>25000</number>
    </property>
    <property name="singleStep">
     <number>1000</number>
    </property>
    <property name="value">
     <number>6000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>meters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Range to lanes (meters)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="laneRange_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>1400</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="laneRange_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>2400</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>N bnd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S bnd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearGraph">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dB_shipStDev">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>130</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dB_shipMean">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>mean dB when ship abeam</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>299</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Transmission Loss:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Near spreading</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QDoubleSpinBox" name="nearSpreading">
       <property name="value">
        <double>18.600000000000001</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>dB/decade in R</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Range to switch</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="2">
      <widget class="QSpinBox" name="rangeCylindrical">
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>4000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>to cylindrical</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>143</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>...Ship min length (m)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="minShipLength">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>180</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>1</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Policies:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="policy_0">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>24</y>
      <width>136</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Random N and S</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="policy_1">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>54</y>
      <width>136</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed limit (kts)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>85</y>
      <width>143</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>...dB reduction per kt</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dBperKt">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>85</y>
      <width>69</width>
      <height>26</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="policy_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>117</y>
      <width>204</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Two convoys/day  AM - PM</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>146</y>
      <width>143</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>...Ship separation (m)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="shipSeparation">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>146</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>600</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="speedLimit">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>53</y>
      <width>69</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>11.000000000000000</double>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="yCaption">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="plainText">
     <string>Percent of Time RL less than X axis value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>610</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Received Level (RL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ave. Convoy</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="convoySize">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConvoy_0_9_Val_Veirs">
    <property name="title">
     <string>Convoy 0.9 Val Veirs</string>
    </property>
   </widget>
   <addaction name="menuConvoy_0_9_Val_Veirs"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
